Applications/Spreadsheet/SpreadsheetView.cpp

```diff
@@ -79,6 +79,12 @@ SpreadsheetView::SpreadsheetView(Sheet& sheet)
     };
 
     m_table_view->on_selection_change = [&] {
+        m_sheet->selected_cells().clear();
+        for (auto& index : m_table_view->selection().indexes()) {
+            Position position {m_sheet->column(index.column()), (size_t)index.row()};
```

Comment:
c l a n g f o r m a t

Reply:
w h y m e r g e i t t h e n aaaaaaaaaa

Reply:
because it was fixed in the next commit aaaaaaaaaaaaaaa

Reply:
nooooooo formatting segmentation bad

Reply:
there should be something like `git clang-format-defrag` that takes each commit, formats it, then amends it with the formatted files